corehq/apps/export/export.py

```diff
@@ -416,6 +419,13 @@ def __record_datadog_export(duration, doc_bytes, n_rows, metric, tags):
         )
 
 
+def _record_export_duration(duration, export_id):
+    # reload the export again to avoid stale updates to pre-loaded export doc
```

Comment:
you could just try save it and then reload if you get a `ResourceConflict` error